bin/run_analysis.py

```diff
@@ -19,16 +23,17 @@ def create_parser():
                         help="the number of batches to train the neural net. 100 is good for a dry "
                         "run, but around 5000 is recommended for statistical significance")
     parser.add_argument("--batch-size",
-                        default=96,
+                        default=100,
                         type=int,
-                        help="around 100 is a good choice, defaults to 96 because cifar10 does")
+                        help="around 100 is a good choice")
     parser.add_argument("--band-list",
                         default=[0, 0, 0, 1],
                         nargs=4,
                         type=int,
                         help="specify which bands to activate (R  G  B  IR). default is "
                         "--bands 0 0 0 1 (which activates only the IR band)")
     parser.add_argument("--render-results",
+                        default=True,
```

Comment:
If you want the results to be rendered by default, it would be better to call the tag something like `--disable-result-rendering` and set it to `action='store_false'`